support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -34,12 +37,14 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     context: Context,
     services: Services
   ): FutureHandlerResult = {
-    val now = () => OffsetDateTime.now
+    val nowJavaTime = () => OffsetDateTime.now
```

Comment:
How much work would it be to change the uses of java time over to Joda (or the other way but I think that would be a lot more work)?

Reply:
just to get an idea -
java to joda would be 7 files
getting rid of joda would be 52 files

It's fairly mechanical in either direction (the lead on the java time project was the joda-time person)
joda has no dependencies so it would Not be a massive artifact size win to remove it
users are officially "asked" to migrate to the Java 8 time apis as it's no longer in development
> Joda-time is no longer in active development except to keep timezone data up to date. From Java SE 8 onwards, users are asked to migrate to java.time (JSR-310) - a core part of the JDK which replaces this project.

my personal opinion if we're going to do either, just get onto the new api in one go as then it's cleaner, but either way not in this PR.